.claude/skills/resend/send-email/references/installation.md

```
@@ -0,0 +1,123 @@
+Add to `mix.exs`:
+```elixir
+def deps do
+  [
+    {:resend, "~> 0.4.0"}
+  ]
+end
+```
+
+## API Key Setup
+
+All SDKs require a Resend API key. Get one at https://resend.com/api-keys
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)